index.html

```diff
@@ -8,50 +8,267 @@
+                        </li>
+                        <li class="icon ">
+                            <a href="https://www.linkedin.com/in/ibrahim-ahmat-b5513b1a6/ " target="”_blank”" rel="noreferrer noopener "><img src="./assets/linkedin.svg " alt="Linked-In icon " /></a>
+                        </li>
+                        <li class="icon ">
+                            <a href="# " rel="noreferrer "><img src="./assets/medium.svg " alt="Medium icon " /></a>
+                        </li>
+                        <li class="icon ">
+                            <a href="https://github.com/ibr5500 " target="”_blank”" rel="noreferrer noopener "><img src="./assets/github.svg " alt="Github icon " /></a>
+                        </li>
+                        <li class="icon ">
+                            <a href="# " rel="noreferrer "><img src="./assets/angellist.svg " alt="Angellist icon " /></a>
+                        </li>
+                    </ul>
+                </div>
+                <div>
+                    <div class="about-button-container flex">
+                        <button type="button" class="button flex">Get my resume</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class='box flex'>
+            <div class="box-items flex">
+                <div class="box-header flex">
+                    <h4>Langguages</h4>
+                    <a href="# "><img src="./assets/angle-down.png " alt="# " /></a>
+                </div>
+                <div class="langs flex">
+                    <div class="lang-box-items">
+                        <div class='lang-box flex'>
+                            <img class="eclipse " src="./assets/Jseclipse.png " alt=" " />
+                            <h5>JavaScript</h5>
+                        </div>
+                        <div class='lang-box flex'>
+                            <img class="eclipse " src="./assets/htmleclipse.png " alt=" " />
+                            <h5>HTML</h5>
+                        </div>
+                        <div class='lang-box flex'>
+                            <img class="eclipse " src="./assets/csseclipse.png " alt=" " />
+                            <h5>CSS</h5>
+                        </div>
+                    </div>
+                </div>
+                <div class="speare ">
+                    <div class="box-angle right">
+                        <a href="#"><img src="./assets/angle-right.png " alt="# " /></a>
+                    </div>
+                    <h4>Frameworks</h4>
 
+                    <hr/>
+                    <div class="box-angle right">
+                        <a href="#"><img src="./assets/angle-right.png " alt="# " /></a>
+                    </div>
+                    <h4>Skills</h4>
+
+                    <hr/>
+                </div>
+            </div>
```

Comment:
- [ ] While doing changes,  The design of this section got mismatched. Kindly maintain the same design as in Original website.
- Your website
![Screenshot from 2022-03-20 00-51-18](https://user-images.githubusercontent.com/26767383/159135576-fccc645a-2d24-41a3-989d-d8e73f0bc370.png)

- Original website 
![Screenshot from 2022-03-20 00-51-27](https://user-images.githubusercontent.com/26767383/159135581-1c91d86b-33ef-45c2-bb7c-28ff9e3971d7.png)